andela/Titan-Devs
# #166027582 Fix average rating returned

#### What does this PR do?
- Fix the average rating returned 
#### Description of Task to be completed?
-  Change the calculation of rating average based on the articleId
#### How should this be manually tested?
- Clone the repository and checkout the branch `bg-fix-average-rating-166027582`
- Run the tests
- Start the server with `yarn start` and create an article 
- Rate an article by consuming the API `POST: /articles/:slug/rating`
- Fetch the ratings of an article by `GET: /articles/:slug/rating`
- You should see the average rating of the article. 
#### Any background context you want to provide?
Initially, when fetching the average rating of an article, we were getting average which is calculated incorrectly. Now the average rating is calculated correctly. 
#### What are the relevant pivotal tracker stories?
#166027582
